src/main/java/org/springframework/cloud/deployer/spi/kubernetes/KubernetesAppDeployer.java

```diff
@@ -226,6 +229,25 @@ public String getLog(String appId) {
 		return logAppender.toString();
 	}
 
+	@Override
+	public void scale(AppScaleRequest appScaleRequest) {
+		logger.debug(String.format("Scale app: %s to: %s",
+				appScaleRequest.getDeploymentId(), appScaleRequest.getCount()));
+
+		ScalableResource scalableResource =
+				(this.client.apps().statefulSets().withName(appScaleRequest.getDeploymentId()).get() != null) ?
```

Comment:
consider just making a local `String deploymentId = appScaleRequest.getDeploymentId()` rather than making the same call multiple times here

Reply:
ok, changed this and also changed the order of checking so that we check for Deployments resource first assuming that the StatefulSet resources will be less common.